### PR TITLE
Add description about HomeWizard Energy Socket "Light level" control

### PR DESCRIPTION
**Pull request by DCSBL**
## Proposed change

Add info about "Light level" for the HomeWizard Energy Socket

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/82378
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by DCSBL** [comment]
Name has to be adjusted when agreed in https://github.com/home-assistant/core/pull/82378#discussion_r1029942977

**Comment by frenck** [comment]
@DCSBL If you can make that last adjustment, we can merge this one in too :)

**Comment by DCSBL** [comment]
Done 👍

File: source/_integrations/homewizard.markdown [hunk 1]
```diff
@@ -65,9 +65,11 @@ The HomeWizard Energy API only exposes properties that are used within the HomeW
...
-## Switches
+## Energy Socket
 
-The Wifi Energy Socket (`HWE-SKT`) outlet state can be controlled the switch platform. There are two switches:
+The Wifi Energy Socket (`HWE-SKT`) outlet state and status light can be controlled. There are two switches:
 
 - **Switch**: Controls the outlet state of the Energy Socket. This switch is locked out when `Switch Lock` is turned on. 
 - **Switch lock**: Forces the outlet state in the `on` position and disables the physical button. This option is useful when the socket is used for a device that must not be turned off, such as a refrigerator.
+
+You can also control the green status light brightness with **Status light brightness**. This light turns on when the `Switch` is on.
```
Comment (frenck) [inline source/_integrations/homewizard.markdown]:
```suggestion
You can also control the green status light brightness with **Status light brightness**. This light turns on when the switch is on.
```

**Review by frenck — APPROVED** [review]
Thanks, @DCSBL 👍

**Pull request merged**